### ISSUE TITLE
New Task: Run ChemDataExtractor on Free Text

### ISSUE THREAD

**Issue by kjappelbaum** [OpenBioML/chemnlp#18]

**Comment by kjappelbaum** [comment]
For better train/test split and LIFT prompting

**Comment by marcosfelt** [comment]
Yes! I have a demo of that here: https://colab.research.google.com/drive/1r7RkBnQ9bE1kY9TTQqbSJk4kfVrt2UKa#scrollTo=KJ7SeCEk7L1l

**Comment by kjappelbaum** [comment]
This might also be useful https://github.com/markmartorilopez/OChemR
